gossipd/gossipd.c

```diff
@@ -310,14 +307,20 @@ static u8 *create_node_announcement(const tal_t *ctx, struct daemon *daemon,
 
 static void send_node_announcement(struct daemon *daemon)
 {
-	u32 timestamp = time_now().ts.tv_sec;
+	u32 timestamp = time_now().ts.tv_sec & ~0x3;
```

Comment:
perhaps a comment for this line

Reply:
Had the same issue, this is just rounding down to `// 3` isn't it? Is this to make room for the +1 and still be lower than now? We don't prohibit future timestamps, so why is this necessary?

Reply:
Oops, this was just to trigger the problem more easily.  Was **NOT** supposed to be in the patch!  Will remove with !fixup